daemon/src/taker_cfd.rs

```diff
@@ -60,6 +60,29 @@ enum RollOverState {
     None,
 }
 
+pub struct AutoRolloverParams {
+    settlement_interval: Duration,
+    /// Defines the minimum duration a CFD needs to be open for until a roll-over is triggered.
+    max_open_until_rollover: Duration,
+}
+
+impl AutoRolloverParams {
+    pub fn new(rollover_interval: Duration, settlement_interval: Duration) -> Result<Self> {
+        Ok(Self {
+            settlement_interval,
+            max_open_until_rollover: rollover_interval,
+        })
+    }
+
+    // Two conditions need to be bet for a roll over to be triggered.
```

Comment:
```suggestion
    // Two conditions need to be met for a roll over to be triggered.
```

or can one use `bet` in this context in English?